composer.json

```diff
@@ -17,10 +17,10 @@
         }
     ],
     "require": {
-        "php": "^5.3.2 || ^7.0",
-        "symfony/framework-bundle": "^2.1 || ^3.0",
```

Comment:
We had support for symfony 2.1 oO?

Reply:
lol :)

Reply:
I guess jordi just tried 2.1 instead 2.3 to get some hints, am I right?

Reply:
oh no, I am wrong, we supported 2.1 oO!